Release Notes/Release Notes v2.9.0.md

```diff
@@ -1,3 +1,30 @@
 # .NET Driver Version 2.9.0 Release Notes
 
-This file is a placeholder.
\ No newline at end of file
+The main new features in 2.9.0 are:
+
+* Sharded transactions using the same API as replica set transactions
+* Sharded transactions pinning to a single mongos router
```

Comment:
This is an implementation detail of the previous bullet item.  No need to mention it separately.

Reply:
Fixed!